### PR TITLE
unarchiving thunderbird autoconfiguration docs

### PR DESCRIPTION
**Pull request by chrisdavidmills**
OK, trying this one again @peterbe

### CODE REVIEW

File: files/en-us/mozilla/thunderbird/autoconfiguration_in_thunderbird/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Autoconfiguration in Thunderbird
+slug: Mozilla/Thunderbird/Autoconfiguration_in_Thunderbird
+tags:
+  - Beginner
+  - Guide
+  - thunderbird
+---
+<p>Moved to <a href="/en/Thunderbird/Autoconfiguration" title="en/Thunderbird/Autoconfiguration">Autoconfiguration</a></p>
```
Comment (peterbe) [inline files/en-us/mozilla/thunderbird/autoconfiguration_in_thunderbird/index.html:9]: Delete this file (`git rm files/en-us/mozilla/thunderbird/autoconfiguration_in_thunderbird/index.html`) and run:
```
yarn tool add-redirect /en-US/docs/Mozilla/Thunderbird/Autoconfiguration_in_Thunderbird /en-US/docs/Mozilla/Thunderbird/Autoconfiguration
```

**Review by peterbe — CHANGES_REQUESTED** [review]
As a reminder, please also run:
```
yarn tool add-redirect /en-US/docs/Thunderbird/Autoconfiguration /en-US/docs/Mozilla/Thunderbird/Autoconfiguration 
```

File: files/en-us/mozilla/thunderbird/autoconfiguration_in_thunderbird/index.html (hunk 1 above)
Comment (chrisdavidmills) [reply files/en-us/mozilla/thunderbird/autoconfiguration_in_thunderbird/index.html]: OK, so all the other redirects worked, except this one, which gave me the error:

`From-URL resolves to a file ($ARCHIVED/en-us/mozilla/thunderbird/autoconfiguration_in_thunderbird/index.html)`

**Comment by chrisdavidmills** [comment]
All the redirects are in place now, except for what I mentioned above.

File: files/en-us/mozilla/thunderbird/autoconfiguration_in_thunderbird/index.html (hunk 1 above)
Comment (peterbe) [reply files/en-us/mozilla/thunderbird/autoconfiguration_in_thunderbird/index.html]: Sigh. We need to merge https://github.com/mdn/archived-content/pull/4 first and then @fiji-flo or I need to update the `archived.txt`. 
But @fiji-flo What is the meaning of this? Why are you no longer allowed to link to an archived page?

Comment (fiji-flo) [reply files/en-us/mozilla/thunderbird/autoconfiguration_in_thunderbird/index.html]: It's linking from an archived page. We check if `FROM` conflicts with an existing document.

Maybe we should move the `archived.txt` to the content repo. Then we could fix that issue by first editing the `archived.txt` when unarchiving.

@peterbe this also uncovered https://github.com/mdn/yari/pull/2750

**Comment by peterbe** [comment]
If we move the `archived.txt` into the `mdn/content` we can make the `unarchive` tool update that file (as well as the actual files it needs to move in). 
But for now, I see now reason to hold this PR back. Right? Revert me if it was wrong to merge :)

**Review by peterbe — APPROVED** [review]

**Pull request merged**